helm/superset/values.yaml

```diff
@@ -357,7 +357,7 @@ postgresql:
   enabled: true
   ##
   ## The name of an existing secret that contains the postgres password.
-  existingSecret:
+  existingSecret: secret-name
```

Comment:
These changes will/can cause unintended consequences.

Reply:
Can we do something like `null` instead?

Reply:
Yes, I a moment I will commit a fix for that.